csi/server/deploy/kubernetes/csi-nodeplugin-opensdsplugin.yaml

```diff
@@ -14,6 +14,9 @@ spec:
       labels:
         app: csi-nodeplugin-opensdsplugin
     spec:
+      tolerations:
```

Comment:
Please add a note that this is only needed for multi-attach.

Reply:
Here means deploying `nodepod` on the master node.